packages/components/src/higher-order/navigate-regions/index.tsx

```diff
@@ -49,7 +49,7 @@ export function useNavigateRegions( shortcuts: Shortcuts = defaultShortcuts ) {
 	function focusRegion( offset: number ) {
 		const regions = Array.from(
 			ref.current?.querySelectorAll< HTMLElement >(
-				'[role="region"][tabindex="-1"]'
+				'[role="region"][tabindex="-1"]:not(:empty)'
```

Comment:
Will do the job, although it's not perfect, since it `:empty` will be false if there's [even just a whitespace character](https://developer.mozilla.org/en-US/docs/Web/CSS/:empty), like a space or a newline:

> Children can be either element nodes or text (including whitespace)

Still, better than before 👍